docs/versioned-plugins/filters/grok-v3.4.1.asciidoc

```diff
@@ -160,33 +160,33 @@ filter. This newly defined patterns in `pattern_definitions` will not be availab
 [id="{version}-plugins-{type}s-{plugin}-options"]
 ==== Grok Filter Configuration Options
 
-This plugin supports the following configuration options plus the <<plugins-{type}s-common-options>> described later.
+This plugin supports the following configuration options plus the <<{version}-plugins-{type}s-{plugin}-common-options>> described later.
```

Comment:
This problem is caused by bad asciidoc in older versions of the plugin. We can fix it with a simple search/replace in the old files.